### ISSUE TITLE
Category-based heuristic

### ISSUE THREAD

**Issue by v3gtb** [v3gtb/fooddata-vegattributes#98]
Part of #9 

"Category" here meaning the FDC-assigned food category of course, not what we call "Category" in our code (vegattributes category).

Because categories are represented totally different in the JSON for Survey and SR Legacy datasets, would be good to do #95 before starting this.

**Comment by v3gtb** [comment]
This doesn't look like it will actually make things significantly more accurate, so I probably won't do it. However, FDC categories are instead very useful for finding wrongly categorized food items manually. See #100.

**Comment by v3gtb** [comment]
Will never be done as explained above
